product_template_tags_code/__manifest__.py

```diff
@@ -0,0 +1,13 @@
+# Copyright 2020 Camptocamp
+# License AGPL-3.0 or later (http://www.gnu.org/licenses/agpl).
+
+{
+    "name": "Product Template Tags",
+    "summary": "This addon allow to add code on products tags",
+    "version": "13.0.1.0.0",
+    "license": "AGPL-3",
+    "author": "Camptocamp SA, Odoo Community Association (OCA)",
+    "website": "https://github.com/OCA/product-attribute",
+    "depends": ["product", "product_template_tags"],
+    "data": ["views/product_template_tag.xml"],
+}
```

Comment:
where is the migration step? Or the post install hook? You should compute codes for existing records.